coded_tools/news_sentiment_analysis/sentiment_analysis.py

```diff
@@ -1,12 +1,8 @@
 import json
 import logging
 import os
-from typing import Any
-from typing import Dict
-from typing import List
-from typing import Tuple
+from typing import Any, Dict, List, Optional, Tuple
```

Comment:
Our convention is to have 1 import per line.
If you run `make lint` the file should be automatically re-formatted.
I wonder why you don't get the right format.

Reply:
`make lint` runs:
```
isort run.py apps/ coded_tools/ --force-single-line
```